Cargo.toml

```diff
@@ -42,6 +42,7 @@ hex = "0.4.2"
 picky = "5.0.0"
 psa-crypto = { version = "0.2.1" , default-features = false, features = ["with-mbed-crypto"], optional = true }
 zeroize = { version = "1.1.0", features = ["zeroize_derive"] }
+picky-asn1-x509 = { version = "0.1.0", optional = true }
```

Comment:
Could you also add it in the related section in the `README.md` file? we try to do this for all dependencies that we use. Tbf I don't know if that still makes sense, but let's keep the habit for now 😄